src/components/Articles.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+import { getArticles } from '../utils/api';
+
+const Articles = () => {
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
+		getArticles().then((articlesFromApi) => {
+			console.log(articlesFromApi.articles);
```

Comment:
If you're confident that this functionality is working then I'd get rid of this console log just to keep things cleaner as your building later functionality :)